app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerActivity.kt

```diff
@@ -73,7 +73,7 @@ class BrainzPlayerActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            ListenBrainzTheme {
+            ListenBrainzTheme(window = window) {
```

Comment:
Why has this been added?

Reply:
Current I couldn't find a Providable Composition Local to access the current window that the composable is being used in.
This parameter is being used to set the status bars and system navigations theming of the given window.